socket/handler_server_list_request.go

```diff
@@ -0,0 +1,31 @@
+package socket
+
+import (
+	"github.com/paroxity/portal/server"
+	"github.com/paroxity/portal/socket/packet"
+)
+
+// ServerListRequestHandler is responsible for handling the ServerListRequest packet sent by servers.
+type ServerListRequestHandler struct{}
+
+// Handle ...
+func (*ServerListRequestHandler) Handle(_ packet.Packet, c *Client) error {
+	var servers []packet.ServerEntry
+
+	var entry packet.ServerEntry
```

Comment:
The requested change being applied here does not make sense and should be reverted.